### ISSUE TITLE
Use Google Maps API for city name lookups

### ISSUE THREAD

**Issue by MartinSeeler** [MartinSeeler/forecast-io-wrapper#3]
It would be nice to call `forecast.getForecastFor("New York City")` if someone don't know the precise coordinates.

**Comment by michelangelocasablancas** [comment]
Yes, I understand that the only way is calling another API (like gmaps) to get the coordinates from the city name?
